src/main/java/com/stripe/model/Review.java

```diff
@@ -15,116 +17,221 @@
     return (this.charge != null) ? this.charge.getId() : null;
   }
 
-  public void setCharge(String chargeId) {
-    this.charge = setExpandableFieldId(chargeId, this.charge);
+  public void setCharge(String id) {
+    this.charge = ApiResource.setExpandableFieldId(id, this.charge);
   }
 
+  /** Get expanded `charge`. */
   public Charge getChargeObject() {
     return (this.charge != null) ? this.charge.getExpanded() : null;
   }
 
-  public void setChargeObject(Charge charge) {
-    this.charge = new ExpandableField<>(charge.getId(), charge);
+  public void setChargeObject(Charge expandableObject) {
+    this.charge = new ExpandableField<Charge>(expandableObject.getId(), expandableObject);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="paymentIntent">
+  /** Get id of expandable `paymentIntent` object. */
   public String getPaymentIntent() {
     return (this.paymentIntent != null) ? this.paymentIntent.getId() : null;
   }
 
-  public void setPaymentIntent(String paymentIntentId) {
-    this.paymentIntent = setExpandableFieldId(paymentIntentId, this.paymentIntent);
+  public void setPaymentIntent(String id) {
+    this.paymentIntent = ApiResource.setExpandableFieldId(id, this.paymentIntent);
   }
 
+  /** Get expanded `paymentIntent`. */
   public PaymentIntent getPaymentIntentObject() {
     return (this.paymentIntent != null) ? this.paymentIntent.getExpanded() : null;
   }
 
-  public void setPaymentIntentObject(PaymentIntent paymentIntent) {
-    this.paymentIntent = new ExpandableField<>(paymentIntent.getId(), paymentIntent);
+  public void setPaymentIntentObject(PaymentIntent expandableObject) {
+    this.paymentIntent =
+        new ExpandableField<PaymentIntent>(expandableObject.getId(), expandableObject);
   }
-  // </editor-fold>
 
-  // <editor-fold desc="approve">
   /**
-   * Approve a review.
+   * Returns a list of <code>Review</code> objects that have <code>open</code> set to <code>true
+   * </code>. The objects are sorted in descending order by creation date, with the most recently
+   * created object appearing first.
    */
-  public Review approve() throws StripeException {
-    return approve(null, null);
+  public static ReviewCollection list(Map<String, Object> params) throws StripeException {
+    return list(params, (RequestOptions) null);
   }
 
   /**
-   * Approve a review.
+   * Returns a list of <code>Review</code> objects that have <code>open</code> set to <code>true
+   * </code>. The objects are sorted in descending order by creation date, with the most recently
+   * created object appearing first.
    */
-  public Review approve(Map<String, Object> params) throws StripeException {
-    return approve(params, null);
+  public static ReviewCollection list(Map<String, Object> params, RequestOptions options)
+      throws StripeException {
+    String url = String.format("%s%s", Stripe.getApiBase(), "/v1/reviews");
+    return requestCollection(url, params, ReviewCollection.class, options);
```

Comment:
9) List (standard): Static method to list resources.

Reply:
https://github.com/stripe/stripe-java/blob/2aa5db5c39dc6e91228507798083f373904cc8c0/src/main/java/com/stripe/model/Review.java#L140-L149